### DISCUSSION TITLE
What measurements available at metrics endpoint?

### DISCUSSION THREAD

**Discussion by ThomasWetzel** [datalust/seq-tickets#1711] [General]
Hi,

in the documentation is the endpoint api/appinstances/{id}/metrics/{measurement} mentioned.

Is there a list of all possible measurements available?

By the way: It would be great to have a swagger UI available to explore the api. :-)

Thomas

**Comment by nblumhardt** [comment]
Hi Thomas!

Agreed on the Swagger point - something long on our list and one we'll hopefully return to soon....

The metrics endpoint supports:

```
                input.arrivedevents
                input.ingestedevents
                input.ingestedbytes
                input.invalidpayloads
                http.unauthorizedpayloads
                querycache.timeslicehits
                querycache.invalidations
                clusternode.bytessent
                clusternode.bytesreceived
```

The two `querycache.*` metrics are unavailable in 2023.1 (the query cache implementation has been pushed much lower in the stack).

Hope this helps,
NIck

**Reply by ThomasWetzel** [reply]
Thank you Nick.

If this could be provided in Prometheus format ...

Merry Christmas :-)
